cli/connect/console.go

```diff
@@ -186,6 +187,12 @@ func getExecutor(console *Console) prompt.Executor {
 			log.Debugf("Failed to append command to history file: %s", err)
 		}
 
+		evalFuncBody, err := static.GetStaticFileContent(ConnectLuaTemplateFS, "eval_func_body.lua")
+		if err != nil {
+			log.Warnf("Failed to get static file content: %s", err)
```

Comment:
jfyi, `log.Fatalf` should be used here. Take a look on executor structure and you will understand it.